packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.exposurepoint;
+
+import android.hardware.camera2.CaptureRequest;
+import android.hardware.camera2.params.MeteringRectangle;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import io.flutter.plugins.camera.CameraProperties;
+import io.flutter.plugins.camera.features.CameraFeature;
+import io.flutter.plugins.camera.features.Point;
+import io.flutter.plugins.camera.features.regionboundaries.CameraRegions;
+import java.util.concurrent.Callable;
+
+/** Exposure point controls where in the frame exposure metering will come from. */
+public class ExposurePointFeature extends CameraFeature<Point> {
+  // Used later to always get the correct camera regions instance.
+  private final Callable<CameraRegions> getCameraRegions;
+  private Point currentSetting = new Point(0.0, 0.0);
+
+  public ExposurePointFeature(
+      CameraProperties cameraProperties, Callable<CameraRegions> getCameraRegions) {
+    super(cameraProperties);
+    this.getCameraRegions = getCameraRegions;
+  }
+
+  @Override
+  public String getDebugName() {
+    return "ExposurePointFeature";
+  }
+
+  @Override
+  public Point getValue() {
+    return currentSetting;
+  }
+
+  @Override
+  public void setValue(@NonNull Point value) {
+    this.currentSetting = value;
+
+    try {
+      if (value.x == null || value.y == null) {
+        getCameraRegions.call().resetAutoExposureMeteringRectangle();
+      } else {
+        getCameraRegions.call().setAutoExposureMeteringRectangleFromPoint(value.x, value.y);
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Please add a comment explaining why it's desirable to silently (from the perspective of calling code) discard all exepctions.

Reply:
Due to a design error the `cameraRegions` object could be `null` causing the possible `NullReferenceException` when `setValue` was called too early. 

I have corrected the design and made sure `cameraRegions` cannot be null and therefore removed the need to catch potential `NullReferenceExceptions`.